### ISSUE TITLE
01a_losses: create docments

### ISSUE THREAD

**Issue by warner-benjamin** [muellerzr/fastai-docment-sprint#6]
Add docments to [01a_losses](https://github.com/muellerzr/fastai-docment-sprint/blob/master/nbs/01a_losses.ipynb).

See the style guide and contribution guide for more details (links will be added when created).

This requires a moderate familiarity PyTorch loss functions, as the methods cover a fastai's implementations of PyTorch loss functions.
